mypy.ini

```diff
@@ -29,5 +35,11 @@ ignore_missing_imports = True
 [mypy-snappy.*]
 ignore_missing_imports = True
 
+[mypy-test.*]
+allow_redefinition = true
```

Comment:
Should we add allow_untyped_globals too?

Reply:
Done